templates/grafana-agent.yaml

```diff
@@ -2,16 +2,16 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: grafana-agent
-  namespace: kube-system
+  namespace: grafana-agent
```

Comment:
so does this mean the namespace doesn't matter?  Can it run in the same namespace as the current juju model?  If so you could templatize it with {juju_model_name} from `self.model.name`